src/storage/src/storage_state.rs

```diff
@@ -869,42 +880,20 @@ impl<'w, A: Allocate> Worker<'w, A> {
     /// Extract commands until `InitializationComplete`, and make the worker
     /// reflect those commands. If the worker can not be made to reflect the
     /// commands, exit the process.
-    ///
-    /// This method is meant to be a function of the commands received thus far
```

Comment:
why'd you remove this part?

Reply:
Have you read it? :D It's copy-pasted from the compute `reconcile` method and pretty much nothing there applies to the storage version.

Reply:
I did read it, but not deeply. And on the surface what it said seemed desirable. 😅